iina/PlayerCore.swift

```diff
@@ -579,17 +580,19 @@ class PlayerCore: NSObject {
     info.disableOSDForFileLoading = true
     guard let path = mpvController.getString(MPVProperty.path) else { return }
     info.currentURL = URL(fileURLWithPath: path)
-    // add files in same folder
-    if ud.bool(forKey: Preference.Key.playlistAutoAdd) {
-      autoLoadFilesInCurrentFolder()
-    }
-    // auto load matched subtitles
-    if let matchedSubs = info.matchedSubs[path] {
-      for sub in matchedSubs {
-        loadExternalSubFile(sub)
+    backgroundQueue.async {
```

Comment:
Any other suggestion? Actually with the improved algorithm it's fast enough for normal situations. Except for some extreme cases, like there're 1000 videos with completely different name and 1000 subs whose names don't contain any video name, (although I think it won't happen,) the algorithm takes about 15s.